code/general/membership.exs

```diff
@@ -0,0 +1,64 @@
+defmodule Membership.NewMapSetMember do
+  def check_membership(range, list, _mapset) do
+    find = Enum.random(range)
+    mapset = MapSet.new(list)
+    MapSet.member?(mapset, find)
+  end
+end
+
+defmodule Membership.MapSetMember do
+  def check_membership(range, _list, mapset) do
+    find = Enum.random(range)
+    MapSet.member?(mapset, find)
+  end
+end
+
+defmodule Membership.EnumMember do
+  def check_membership(range, list, _mapset) do
+    find = Enum.random(range)
+    Enum.member?(list, find)
+  end
+end
+
+defmodule Membership.EnumAny do
+  def check_membership(range, list, _mapset) do
```

Comment:
I'm not sure this is fair to include in this comparison since `Enum.any?/2` is a much more flexible function than the others that we're comparing. It will almost always be slower since in the majority of use cases it's testing for more than just strict equality (i.e. comparing with `Kernel.===/2` like all the other functions do).